action.yaml

```diff
@@ -182,3 +190,26 @@ runs:
         BAZEL_NEGATIVE_SCOPE_FILTER: ${{ inputs.test-negative-scope-filter }}
         BAZEL_NEGATIVE_TAG_FILTER: ${{ inputs.test-negative-tag-filter }}
         CI: "true"
+
+    - name: Test Impacted Targets (with debugger)
+      if: inputs.test-targets == 'true' && inputs.enable-trunk-ci-debugger
+      id: test-impacted-targets-dbg
+      uses: trunk-io/breakpoint@v1
+      with:
+        breakpoint-id: ${{ inputs.trunk-ci-debugger-breakpoint-name }}
+        trunk-token: ${{ env.TRUNK_TOKEN }}
```

Comment:
Don't we usually specify to use this from `secrets.`?

Reply:
Yes, but an action can't access secrets from the og repo, maybe I'll just add it as in input

Reply:
Actually completely missed that the action already takes a token as an input, modified